pxb-tag/package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "@pxblue/tag",
+    "description": "A script that can be used to automatically tag packages in a continuous integration pipeline.",
+    "version": "1.0.0",
+    "author": "Power Xpert Blue <pxblue@eaton.com> (https://github.com/pxblue)",
+    "license": "BSD-3-Clause",
+    "repository": {
+        "url": "https://github.com/pxblue/automation",
+        "type": "git"
+    },
+    "bugs": {
+        "url": "https://github.com/pxblue/automation/issues"
+    },
+    "keywords": [
+        "pxblue",
+        "automation",
+        "tag",
+        "git"
+    ],
+    "main": "index.js",
+    "bin": {
+        "pxb-tag": "scripts/tag.sh"
+    },
+    "files": [
+        "scripts/",
+        "README.md",
+        "CHANGELOG.md",
+        "LICENSE",
+        "package.json"
+    ],
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 0",
+        "pxb-tag": "pxb-tag"
```

Comment:
Is this necessary? Can't you just call the command directly (i.e., not with yarn) since it's defined in bin?

Reply:
That's what I thought too, but for some unknown reason I cannot call the `pxb-tag` command directly, even though it's in the node_modules bin.  It works in the context of a yarn command, but it doesn't work if you call it directly. 

This is the error I ran into: https://app.circleci.com/pipelines/github/pxblue/automation/85/workflows/37a40bbc-4bc4-46f6-b8bf-85ed20c41440/jobs/111

Reply:
Nevermind I got this to work.